app/actions/client.py

```diff
@@ -213,8 +217,38 @@ async def get_station_conditions(integration, base_url, config, auth):
                         message=parsed_response["message"],
                         status_code=parsed_response["code"]
                     )
-                obs = ConditionsResponse.parse_obj(parsed_response)
-                return obs
+
+                if any("fault_status" in cond for cond in parsed_response.get("conditions", [])):
```

Comment:
@vgarcia13 are you confident this will only exclude and report for non-zero fault_status conditions?

I also recommend moving this filter logic and activity logger out of the client and into the hander. The general principal being that the client should be concerned with requesting data from the service and returning data to the action runner.

Placing the filtering logic in the action runner also prevents you from having to pass a Gundi integration object into the client.

One other benefit will be that if a user decides to do something with the non-zero fault status conditions, then we will easily add that code to the action runner.